src/index.ts

```diff
@@ -79,10 +79,31 @@ async function main() {
   if (isPreRelease) {
     debug(`pre-release: ${latest.clean}`);
     let preRelease = semver.prerelease(latest.clean);
-    if (preRelease && preRelease.join(".") === "pre.0") {
```

Comment:
Because there was no else block here, prereleases beyond 0 would use the default `previous` value which is just the second entry in `gitTags`, and `INCLUDE_NIGHTLY` defaults to true so it would pick up the prior nightly if it existed.

Another option is to set `INCLUDE_NIGHTLY` only if the release being commented for is a nightly, but that likely involves a check in a github action express which are just a bit harder to test/debug/etc.

Prereleases should never compare against nightlies anyway so it seems fine to ignore nightlies in this flow regardless of the value of `INCLUDE_NIGHTLY`